app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="sign-in-up border-black">
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: "Username", class: "form-fields border-black form-title" %>
+  </div>
+
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email", class: "form-fields border-black form-title" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] [OPTIONAL] Great job so far @vanelnw 👍🏽 , However, as stated by the project requirements, you need to modify your design to match the wireframe provided. Please also take note that I could not signup despite the fact that I provided all the credentials needed!
- It is very important that you refactor your code to achieve this.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="602" alt="Screenshot 2023-03-09 at 19 53 09" src="https://user-images.githubusercontent.com/58692590/224129657-fe14042b-dad2-406f-ad0a-fcdf07b762a0.png"> | <img width="443" alt="Screenshot 2023-03-09 at 19 51 38" src="https://user-images.githubusercontent.com/58692590/224129695-6058e015-f2fa-40b2-8939-9c99201a10bf.png">

Reply:
hi @Whoistolu 
Thank you for mentioning all these amelioration, i will work on it